From f7aec12d5eb2a0b884b35fa0ee7d9715464c84c0 Mon Sep 17 00:00:00 2001
From: Jason Schwanz <jschwanz@users.noreply.github.com>
Date: Thu, 9 Jul 2015 13:58:00 -0700
Subject: [PATCH 03/17] Update keepalived.spec.in

@VERSION@ is not valid in the spec file, replaced with the output of the VERSION file
---
 keepalived.spec.in | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/keepalived.spec.in b/keepalived.spec.in
index 00cf0f2..ccaa11e 100644
--- a/keepalived.spec.in
+++ b/keepalived.spec.in
@@ -4,9 +4,12 @@
 # Ugly, but we need headers from a kernel to rebuild against
 %define kernel %(rpm -q kernel-devel --qf '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}\\n' 2>/dev/null | head -1)
 
+# read in the version from the VERSION file
+%define version %(cat ./VERSION)
+
 Summary: HA monitor built upon LVS, VRRP and services poller
 Name: keepalived
-Version: @VERSION@
+Version: %{version}
 Release: 5
 License: GPL
 Group: Applications/System
-- 
2.4.6

